src/instance/methods/layer.js

```diff
@@ -17,8 +16,8 @@ import { setWithCredentials } from 'ol/featureloader';
 import * as Style from 'ol/style';
 import colorStyles, { clusterStyle } from '../../styles';
 
-// Import the default projection configuration
-import projection from '../../projection';
+// Import readFeatures function.
+import readFeatures from './features';
```

Comment:
Nitpick: Seems like the new import is pretty self-explanatory compared with the old one. Maybe we don't need the comment at all now?

Reply:
Yea I thought so too - but I'm a stickler for consistency, and all other imports are commented, so I'd rather include it even if it doesn't add much. :-)